infrastructure/helm-chart/templates/ingress-ui.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
     traefik.frontend.rule.type: PathPrefixStrip
 spec:
   rules:
-    - host: {{ get (urlParse .Values.global.kubernetes.host) "host" }}
+    - host: {{ .Values.global.kubernetes.host }}
```

Comment:
In the `airy.yaml` file now we specify only the hostname and the protocol (http or https) is specified in `ingress.https` (true or false).